bin/ci/ubuntu/install-dependencies.sh

```diff
@@ -25,6 +25,19 @@ if [[ -n ${CLANG_VER:-} ]]; then
     export LLVM_CONFIG="llvm-${LLVM_VERSION}/bin/llvm-config";
 fi
 
+if [[ ${BUILD:-} == cmake ]]; then
+    # There are cases where the directory exists, but the exe is not available.
+    # Use this workaround for now.
+    if [[ ! -x cmake/bin/cmake && -d cmake ]]; then
+        rm -fr cmake
+    fi
+    if [[ ! -d cmake ]]; then
+      CMAKE_URL="http://www.cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz"
```

Comment:
Please use `https` URLs and don't use `--no-check-certificate` unless it's 100%, absolutely unavoidable.